src/core/server/config/__snapshots__/env.test.ts.snap

```diff
@@ -116,6 +118,7 @@ Env {
     "branch": "some-branch",
     "buildNum": 9007199254740991,
     "buildSha": "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "dist": undefined,
```

Comment:
__note:__ hmm, can we make sure `isKibanaDistributable` and `dist` are always `boolean`? Just for the sake of consistency.

Reply:
agree, TS fooled me
![2019-07-05_12-47-19](https://user-images.githubusercontent.com/3198181/60717819-41e96f80-9f23-11e9-811b-ed8baba487ce.png)